### ISSUE TITLE
Missing some request using GRPC client with error "OS Error"

### ISSUE THREAD

**Issue by CS-savvy** [tensorflow/serving#1464]
I am serving a faster R-CNN model in TensorFlow serving(CPU only) pulled from docker Hub with the latest tag.
it is working fine in most of the cases but from time to time( 1 out of 10 request) I am getting GRPC client error. It is difficult to reproduce this issue cause it is not happening all the time.  

The error I am getting - 

```
<_Rendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "OS Error"
	debug_error_string = "{"created":"@1571383682.087580699","description":"Error received from peer","file":"src/core/lib/surface/call.cc","file_line":1017,"grpc_message":"OS Error","grpc_status":14}"
>
```

**Platform information** - 
Azure VM with 
32 GB Ram 
CPU -  Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz ( 8 cores)
OS - Ubuntu 16.04.6 LTS
Docker - Docker version 18.09.7, build 2d0083d

Tf-serving information -

Pulled from docker hub - [This one](https://hub.docker.com/layers/tensorflow/serving/latest/images/sha256-f7e59a29cbc17a6b507751cddde37bccad4407c05ebf2c13b8e6ccb7d2e9affb)
Build using - TF_SERVING_VERSION_GIT_BRANCH=1.14.0

GRPC client Call method - 
language used - python
tensorflow-serving-api version - 1.13.0

```
from tensorflow_serving.apis import predict_pb2
from tensorflow_serving.apis import prediction_service_pb2_grpc
from tensorflow.python.framework.tensor_util import make_tensor_proto , MakeNdarray
channel = implementations.insecure_channel(settings.TF_SERVING_IP, settings.TF_SERVING_PORT)
stub = prediction_service_pb2_grpc.PredictionServiceStub(self.channel._channel)
request = predict_pb2.PredictRequest()
request.model_spec.name = "model_name"
request.model_spec.signature_name = 'PREDICT'
request.inputs['input_img'].CopyFrom(make_tensor_proto(img_array[None], dtype='float32'))
stub.Predict(request, 120)

```

Also, can someone tell me how many concurrent requests tf-serving can handle without any error?

**Comment by rmothukuru** [comment]
Make sure you also include the exact command if possible to produce the output included in your test case. If you are unclear what to include see the issue template displayed in [the Github new issue template](https://github.com/tensorflow/tensorflow/issues/new/choose).

We ask for this in the issue submission template, because it is really difficult to help without that information. Thanks!

**Comment by gowthamkpr** [comment]
This might be happening mainly due to the absence of network access. 

Please take a look at this [issue](https://github.com/grpc/grpc/issues/16515) and let me know if you are able to solve your issue. Thanks!

**Comment by CS-savvy** [comment]
@rmothukuru  - It is very difficult to reproduce cause it is not happening every time only sometimes it fails to get results. I have already defined my VM configuration and the GRPC client code I am using.

Also, I have realized that I was using tensorflow/serving:1.12.3-gpu docker image in CPU machine,
may be that is the problem, Now I have switched to tensorflow/serving:latest. I have not found any error till now, I will re-open this issue if I will get one.
Thanks for your time.
